test/extended/router/gatewayapicontroller.go

```diff
@@ -539,3 +542,15 @@ func assertHttpRouteSuccessful(oc *exutil.CLI, gwName, name string) (*gatewayapi
 	e2e.Logf("httpRoute %s/%s successful", namespace, name)
 	return checkHttpRoute, nil
 }
+
+// Check for the existence of the okd-scos string in the version name to determine if it is OKD
+func isOKD(oc *exutil.CLI) bool {
+	current, err := exutil.GetCurrentVersion(context.TODO(), oc.AdminConfig())
+	if err != nil {
+		return false
+	}
```

Comment:
Would it be better to return the error and fail the test in case of an error?  I think it would be appropriate at least to log the error.

Reply:
done. fails in case fetching the clusterversion errors out.